views/js/GoogleAnalyticActionLib.js

```diff
@@ -105,6 +105,7 @@ var GoogleAnalyticEnhancedECommerce = {
 		ga('ec:setAction', 'refund', {
 			'id': Order.id // Transaction ID is only required field for full refund.
 		});
+		ga('send', 'event','Refund','refund',{'nonInteraction':1});
```

Comment:
Regarding the [Google Analytics documentation](https://developers.google.com/analytics/devguides/collection/analyticsjs/enhanced-ecommerce#measuring-refunds):
```js
ga('send', 'event', 'Ecommerce', 'Refund', {'nonInteraction': 1});
```